src/rgw/services/svc_sys_obj_cache.cc

```diff
@@ -311,11 +311,9 @@ int RGWSI_SysObj_Cache::write(const rgw_raw_obj& obj,
     // call distribute_cache, as a) it's unclear whether other RGWs
     // will need that system object in the near-term and b) it
     // generates additional network traffic.
-    if (!exclusive) {
-      int r = distribute_cache(name, obj, info, UPDATE_OBJ, y);
-      if (r < 0)
-	ldout(cct, 0) << "ERROR: failed to distribute cache for " << obj << dendl;
-    }
+    int r = distribute_cache(name, obj, info, UPDATE_OBJ, y);
+    if (r < 0)
+      ldout(cct, 0) << "ERROR: failed to distribute cache for " << obj << dendl;
```

Comment:
please remove or update the comment above that describes why it didn't distribute for exclusive creates

Reply:
I removed the comment